crates/gosub-bindings/src/lib.rs

```diff
@@ -91,12 +90,10 @@ pub unsafe extern "C" fn gosub_render_tree_next_node(
 /// and a mutable pointer owned by the C API to write (copy) the contents
 /// of the read-only pointer into the mutable pointer.
 #[no_mangle]
-pub unsafe extern "C" fn gosub_render_tree_get_node_data(
-    node: *const Node,
-    node_data: *mut CTextNode,
-) {
+pub unsafe extern "C" fn gosub_render_tree_get_node_data(node: *const Node, c_node: *mut CNode) {
```

Comment:
new struct `CNode` introduced to more properly wrap the `node_t` struct from the C API. This is more generic than the previous `CTextNode` when we add more types